core/src/main/scala/org/apache/spark/sql/TiStrategy.scala

```diff
@@ -80,6 +80,10 @@ case class TiStrategy(getOrCreateTiContext: SparkSession => TiContext)(sparkSess
   private def allowIndexRead(): Boolean =
     sqlConf.getConfString(TiConfigConst.ALLOW_INDEX_READ, "true").toLowerCase.toBoolean
 
+  private def allowTiFlashRead(): Boolean = {
```

Comment:
`{}` is not neccesary, just make it of similar format in this `allow*` member category.